CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to Transcriptase
```

Comment:
I just noticed that this says the name of the app is `Transcriptase`. You should change it to "Pomodoro"

```suggestion
# Contributing to Pomodoro
```